dist/package/openSUSE-release-tools.spec

```diff
@@ -298,6 +310,14 @@ exit 0
 %postun check-source
 %systemd_postun
 
+%pre docker-publisher
+getent passwd opensuse-relmgr-bot > /dev/null || \
```

Comment:
The pattern is `osrt-docker-publisher`. When deployed move away from any existing users on packagelists which also works well during migration period where things run half and half.

Reply:
I don't quite understand - this .spec file is only useful for packagelists AFAICT and we already agreed on `opensuse-relmgr-bot` for this on packagelists

Reply:
This spec produces packages deployed on 4 machines and part of the point was to standardize names since users, services, and everything else were all over the place before.

Reply:
Ok - then I'm not sure why @DimStar77 and @lnussel made this proposal...

Once this is deployed I guess I'll move it over?